packages/mask/src/extension/popups/modals/modals.ts

```diff
@@ -4,3 +4,5 @@ import type { ConnectSocialAccountModalOpenProps } from './ConnectSocialAccountM
 
 export const DisconnectModal = new SingletonModal<DisconnectModalOpenProps>()
 export const ConnectSocialAccountModal = new SingletonModal<ConnectSocialAccountModalOpenProps>()
+
+export * from './ChooseNetworkModal/index.js'
```

Comment:
Please don't forget use `SingletonModal`.

Reply:
No, it's not a SingletonModal but a modal controlled by router.